drivers/windows/overlay/ov_endpoint_windows.go

```diff
@@ -271,11 +271,11 @@ func (d *driver) EndpointOperInfo(nid, eid string) (map[string]interface{}, erro
 }
 
 func endpointRequest(method, path, request string) (*hcsshim.HNSEndpoint, error) {
-	if windowsBuild == 14393 {
+	if windowsBuild == osversion.RS1 {
 		endpointMu.Lock()
 	}
 	hnsresponse, err := hcsshim.HNSEndpointRequest(method, path, request)
-	if windowsBuild == 14393 {
+	if windowsBuild == osversion.RS1 {
```

Comment:
Same here;

```suggestion
	if windowsBuild < osversion.RS3 {
```

Reply:
This should match what you do above.
So: 
if windowsBuild < osversion.RS5 {